### ISSUE TITLE
Font Issue in Appium Desktop Application - Square Boxes

### ISSUE THREAD

**Issue by ronjonesjr** [appium/appium-desktop#651]
## Appium or Appium Desktop?

You are reporting an issue at the Appium Desktop repository. Appium Desktop is a wrapper around Appium. If you are having trouble running tests, it is much more likely that the problem you are encountering is not a problem with Appium Desktop but with Appium. For that reason we require you to have tried your testcase with Appium CLI before reporting issues. Check the checkbox below to confirm that you have proven your issue does _not_ reproduce on Appium itself:

- [x ] I have verified that my issue does not occur with Appium and should be investigated as an Appium Desktop issue

## The problem

The icons do not appear properly in Appium Desktop.  This has been happening for several versions now, but specifically, the screen shot is the latest version (v 1.7.1).

![image](https://user-images.githubusercontent.com/20243630/46354833-8ca12880-c62d-11e8-80e9-1e1446e43368.png)

## Environment

- I am running Appium Desktop version _1.7.1_.
- I am on (pick one):
    - [ ] Mac
    - [x ] Windows
    - [ ] Linux

**Comment by dpgraham** [comment]
Can you open the dev tools (right click anywhere on the UI and select `Inspect Element`) and check if there's assets not being loaded properly?

**Comment by ronjonesjr** [comment]
You're right.  The two font URI's seem to be blocked.  Have these URI's recently changed?  It looks like it's the firewall on my side so I'll close this issue for now.

https://at.alicdn.com/t/font_zck90zmlh7hf47vi.woff
https://at.alicdn.com/t/font_zck90zmlh7hf47vi.ttf

**Comment by dpgraham** [comment]
@ronjonesjr Possibly, but that's still a problem with Appium Desktop. We shouldn't be downloading fonts, the fonts should be bundled with the application.

**Comment by ronjonesjr** [comment]
Interesting, I was just about to reopen it because the fonts do show up in the debug window.  I think it's likely that they are bundled but the URI referencing them should likely be pointing to something local.
